src/main/java/uk/gov/hmcts/reform/professionalapi/configuration/LaunchDarklyConfiguration.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.hmcts.reform.professionalapi.configuration;
+
+import com.launchdarkly.sdk.server.LDClient;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.InterceptorRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+import uk.gov.hmcts.reform.professionalapi.service.impl.FeatureToggleServiceImpl;
+import uk.gov.hmcts.reform.professionalapi.util.FeatureConditionEvaluation;
+
+@Configuration
+public class LaunchDarklyConfiguration implements WebMvcConfigurer {
+
```

Comment:
there is already one class present WebConfig which is also implements WebMvcConfigurer so please rename that class to LaunchDarklyConfiguration and add the configuration over there instead of creating new class.

Reply:
I think this is not valid...We have keep both seperate.